04portfolio-assignment/scripts/blogView.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+// create object for all of the functions that control what blog articles you see
+var blogview = {};
+
+// Populate filters with the authors and categories from the blogsource
```

Comment:
I don't see any actual filters in your blog. If you are using them, you don't need this code....